.github/workflows/pipeline.yml

```
@@ -0,0 +1,221 @@
+          
+          # Instalar Docker Compose si no está instalado
+          if ! command -v docker-compose &> /dev/null; then
+            echo "Instalando Docker Compose..."
+            sudo curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo chmod +x /usr/local/bin/docker-compose
+          fi
+          
+          # El archivo .env para Docker Compose ya está configurado en el servidor
+          echo "Verificando configuración de Docker Compose..."
+          cat .env
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)